src/pages/assessment/Assessment.tsx

```
@@ -0,0 +1,408 @@
+  };
+
+  const handleUpdateAssessment = (assessmentId: string): void => {
+    const assessment = assessments.find((a) => (a as any).id === assessmentId);
+    if (assessment) {
+      setSelectedAssessment(assessment);
+      setIsUpdateModalOpen(true);
+    }
+  };
+
+  const handleDeleteAssessment = (assessmentId: string): void => {
+    const assessment = assessments.find((a) => (a as any).id === assessmentId);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)